src/util.js

```diff
@@ -276,6 +281,85 @@ function copyDescriptors(from, to, filter) {
     }
 }
 
+global.eventsToHandlers = global.eventsToHandlers ? global.eventsToHandlers :
+    {};
+// This function should not be used for methods that return a value
+// Using this function implies small performance overhead, should be used only
+// for debug methods
+function hookTo(prototypeObject, methodName, extension) {
+    var existingMethodImpl = prototypeObject[methodName];
+    var extensions = null;
+    if (typeof existingMethodImpl === "function") {
+        extensions = global.eventsToHandlers[methodName];
+        if (!extensions) {
+            extensions = global.eventsToHandlers[methodName]
```

Comment:
Two reasons to go with local instrumentation:
1) There are name compatibility issues
2) Users would have to handle multiple event handlers themselves (all the functionality of hookTo unhookFrom functions in utils.js) and in this case unsubscribing from the event would be a problem for them.